src/type.ts

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable */
```

Comment:
Why do you disable eslint on this file? Please remove it and let the lint run.

Reply:
Thanks for your review.
For data types that have not yet been determined, any type was used.
It caused an eslint warning.

Reply:
@Dinisueldanov we shouldn't disable eslint for anything actually, if you have to do it means something is going wrong. Please find a way to fix the issue without disabling esLint